riscv/socketif.cc

```diff
@@ -0,0 +1,77 @@
+// See LICENSE for license details.
+
+#include "socketif.h"
+
+#ifdef HAVE_BOOST_ASIO
+
+#include <iostream>
+
+socketif_t::socketif_t()
+{
+  try
+    { // create socket server
+      using boost::asio::ip::tcp;
+      io_service_ptr = new boost::asio::io_service;
+      acceptor_ptr = new tcp::acceptor(*io_service_ptr, tcp::endpoint(tcp::v4(), 0));
+      // acceptor is created passing argument port=0, so O.S. will choose a free port
+      std::string name = boost::asio::ip::host_name();
+      std::cout << "Listening for debug commands on " << name.substr(0,name.find('.'))
+                << " port " << acceptor_ptr->local_endpoint().port() << " ." << std::endl;
+      // at the end, add space and some other character for convenience of javascript .split(" ")
+    }
+  catch (std::exception& e)
+    {
```

Comment:
Looks like the brace formatting in these blocks got screwed up.  Should change it to match the other blocks in this file, which conform to our house style.